src/App.tsx

```diff
@@ -15,36 +16,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface SortFieldAndIsReversed {
+  sortField: string;
+  isReversed: boolean;
+}
+
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+const SORT_FIELD_LENGHT = 'lenght';
+
+function getGoodsPrepared(
+  goods: string[],
+  { sortField, isReversed }: SortFieldAndIsReversed,
+) {
+  let preparedGoods = [...goods];
+
+  if (isReversed) {
+    preparedGoods = preparedGoods.reverse();
+  }
```

Comment:
```suggestion
  if (isReversed) {
    preparedGoods = preparedGoods.reverse();
  }
```
move it after the sorting